develop/CICD/GitHub/GitHub_Starter_Workflows.md

```diff
@@ -0,0 +1,101 @@
+---
+uid: github_starter_workflows
+---
+
+# GitHub starter workflows
+
+We offer a number of starter workflows that can be used as the starting point to create a workflow for your GitHub repository. These .yml files are designed to be small and call both reusable workflows and actions from the marketplace that always try to run the latest versions.
+
+A [GitHub action](xref:Deploying_Automation_scripts_from_a_GitHub_repository) provides an easy access point to a single key action, such as deploying a DMAPP, creating a DMAPP, running tests, and more.
+
+[GitHub reusable workflow](xref:github_reusable_workflows) allows the reuse of a combination of many different GitHub actions and other scripts, running across several jobs combined into a single easy call.
+
+By combining these three concepts, you can avoid maintenance costs and ensure that you always run the latest stable versions without the need for further user configuration. Any default behavior can easily be overwritten and changed on your copy of the starter workflow if you wish to always use a specific version until it is manually adjusted. This might be necessary to meet security requirements for your organization.
+
+## Using a starter workflow
+
+### In a public repository
+
+In your GitHub repository, navigate to the *Actions* tab.
+
+If this is your first workflow, scroll down to the workflows provided by your organization and select the one that matches the content and intent of your repository.
+
+If you want to add a new workflow to your repository, you should see a button *New Workflow* to the left. This will open the starter workflow selection screen.
+
+### In a private repository
+
+If your GitHub organization has the enterprise edition, you will find the starter workflow offered by your organization directly in the UI as explained [above](#in-a-public-repository).
+
+In the free version of GitHub, you will not see starter workflows in the UI directly. In this case, perform the following steps:
+
+1. Navigate to the *workflow-templates* folder in the *.github* repository of your organization.
+
+   For members of Skyline Communications, this is <https://github.com/SkylineCommunications/.github/tree/main/workflow-templates>.
+
+1. Select the workflow template that matches the content of your repository and look at the source code.
+
+1. Copy that and navigate to your own Repository Action tab.
```

Comment:
Is "Repository Action" the exact name of the tab? Or is it "Actions" as mentioned above?

Reply:
"Actions"